### DISCUSSION TITLE
Συνάντηση για καθορισμό webring

### DISCUSSION THREAD

**Discussion by atsiano81** [sdy60-2022/.github#31] [General]
Παρακαλώ να θέσουμε προτάσεις για συνάντηση με σκοπό τον ορισμό webring ώστε να ολοκληρωθεί το πρώτο μέρος

**Comment by ghost** [comment]
θεωρω οτι πλεον ειναι αργα για να κανουμε κατι αλλο εκτος απο το ετοιμο που προτεινεται στην εκφωνηση. ας εγκατασταθει να τελειωνουμε

**Comment by ghost** [comment]
το σαιτ που χρησιμοποιει το υποδειγμα βρισκεται στο repo
https://github.com/maxboeck/webring
ας το κανουμε fork
μετα ο αλεξανδρος θα πρεπει να κανει καποιες ρυθμισεις για να παιζει και στο netlify (πιθανον να παιζει και στο github pages, δε γνωριζω)
ακολουθως πρεπει να κανουμε ολοι επεξεργασια στο webring/src/data/members.json και να προσθεσουμε το δικο μας cv
και τελος να κανουμε push (και pull request) στο repo του οργανισμου

συμφωνουμε;

**Comment by stdi** [comment]
Work done
=========

1. repo forked
2. @NikosAndreadakis added another choice on sdy60 repo for webring implementation
3. netlify connection is missing currently
4. @panoskam has admin access on the forked repo and can manage issues and PRs
